api/app/controllers/api/v1/instructors_controller.rb

```diff
@@ -14,30 +14,44 @@ def index
             render_success(instructors_by_position)
         end
 
-        # POST /add_instructor
+        # POST /instructors AND /add_instructor
         def create
-            position = Position.find(params[:position_id])
-            instructor = position.instructors.new(instructor_params)
-            if instructor.save # passes Instructor model validation
-                render_success(instructors_by_position)
+            # if we passed in an id that exists, we want to update
+            if params[:id] && Instructor.exists?(params[:id])
+                update and return
+            end
+
+            if not params.include?(:position_id)
```

Comment:
To get this straight, if there isn't a position_id in the parameters, we would create a new instructor.

If there is a position_id in the parameters, then we are adding an instructor to that position.

Is that correct?

Reply:
It more like if there isn't a `position_id`, we are creating a new `instructor` and adding that new `instructor` into `position`.

Reply:
If there is no position_id, how can you add an instructor to the position since you don't know the `position_id`?

Reply:
Sorry. I just realized that I made a mistake in my reply. I was referring to when there **is** a `position_id`.